Python/marshal.c

```diff
@@ -252,47 +248,102 @@ w_short_pstring(const void *s, Py_ssize_t n, WFILE *p)
 static PyObject *
 _PyMarshal_WriteObjectToString(PyObject *x, int version, int allow_code);
 
+#define _r_digits(bs)                                                     \
```

Comment:
```suggestion
/* 'bs' is used as `uint{bs}_t` */
#define _r_digits(bs)                                                     \
```

It's a bit hard to read the `uint##bs##_t` the first time.

Reply:
`bitsize` or `bit_size`?

Reply:
Up to you! I would use bitsize and use `uint ## bitsize ## _t` (with whitespaces) as it would read better than `uint ## bit_size ## _t` IMO (less underscores)